server/data/licenceApiClient.ts

```diff
@@ -89,7 +90,7 @@ export default class LicenceApiClient extends RestClient {
     }
   }
 
-  async createLicence(licence: CreateLicenceRequest, user: User): Promise<LicenceSummary> {
+  async createLicence(licence: CreateLicenceRequest, user: User): Promise<LicenceCreationResponse> {
```

Comment:
This is a cut down representation that only includes the licenceId rather than the full object - which is the only bit we actually use